spec/fixtures/invalid-typechecked/tslint.json

```diff
@@ -0,0 +1,5 @@
+{
+  "rules": {
+    "no-unused-variable": true
```

Comment:
Fixed in https://github.com/AtomLinter/linter-tslint/pull/157/commits/f663517f416cd0f9c68d9c3c54c6e113be83b783